Services/CO.CDP.Forms.WebApi/UseCase/UpdateFormSectionAnswersUseCase.cs

```diff
@@ -23,8 +23,52 @@ public async Task<bool> Execute((Guid formId, Guid sectionId, Guid answerSetId,
         var section = await formRepository.GetSectionAsync(formId, sectionId)
             ?? throw new UnknownSectionException($"Unknown section {sectionId} in form {formId}.");
 
-        var sharedConsent = await formRepository.GetSharedConsentDraftWithAnswersAsync(formId, organisationId)
-                            ?? CreateSharedConsent(organisation, section.Form);
+        var sharedConsent = await formRepository.GetUntrackedSharedConsent(formId, organisationId);
```

Comment:
Treat it as optional, but i think all these conditions below could just be reduced to:


```csharp
        var sharedConsent = await formRepository.GetUntrackedSharedConsent(formId, organisationId);
        if (sharedConsent.SubmissionState == Persistence.SubmissionState.Submitted)
        {
            CleanSharedConsent(sharedConsent);
        }
        else
        {
            sharedConsent = await formRepository.GetSharedConsentDraftWithAnswersAsync(formId, organisationId)
                                           ??  CreateSharedConsent(organisation, section.Form);
        }
        await UpdateOrAddStandardAnswers(formRepository, answerSetId, answers, section, sharedConsent);
        await formRepository.SaveSharedConsentAsync(sharedConsent);
```

Reply:
done.